xbmc/settings/windows/GUIWindowTestPattern.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "GUIWindowTestPattern.h"
+
+#include "ServiceBroker.h"
+#include "guilib/GUIMessage.h"
+#include "guilib/GUITexture.h"
+#include "guilib/WindowIDs.h"
+#include "input/Key.h"
+#include "utils/Color.h"
+#include "utils/Geometry.h"
+#include "utils/Variant.h"
+#include "windowing/WinSystem.h"
+#include "xbmc/Application.h"
+
+#include <iostream>
+
+
+CGUIWindowTestPattern::CGUIWindowTestPattern(void) : CGUIWindow(WINDOW_TEST_PATTERN, "")
+{
+
```

Comment:
style request, please remove extra newline here, and use a single newline after `#include <iostream>`